src/languages/dns.js

```diff
@@ -1,14 +1,14 @@
 /*
 Language: DNS Zone
 Author: Tim Schumacher <tim@datenknoten.me>
+Contributors: Dorian CHECCONI <contact@dorianchecconi.fr>
```

Comment:
We don't add individual small contributors in comment - preferring to instead trust GitHub to capture this information.